doc/classes/Navigation.xml

```diff
@@ -85,11 +94,13 @@
 			<argument index="1" name="xform" type="Transform">
 			</argument>
 			<description>
+				Associates a [code]NavigationMesh[/code]'s id with a [code]Transform[/code]. Its position, rotation and scale are are based on the [code]Transform[/code] passed.
```

Comment:
> are are

Reply:
Are are matey, take a long walk down a short gang plank! :D

Reply:
Both should be good now @Noshyaar. Thanks for your feedback :)